### ISSUE TITLE
Incorrect slope test (?)

### ISSUE THREAD

**Issue by bpbond** [PNNL-TES/bahn-rs-test#5]
I see a lot of places where you test for slope=1, I think, e.g.:

`
slope_test <- summary( m1_MAT )$coefficients[ 2, 4 ]
`

But this extracts (from the model summary object) the test for whether the slope is different from _zero_, not from one.

**Comment by bpbond** [comment]
A couple different ways to do this are given [here](https://stats.stackexchange.com/questions/111559/test-model-coefficient-regression-slope-against-some-value).

**Comment by jinshijian** [comment]
Yes, that's true, use this method it actually tests whether slope differs from zero. I said 1 because it makes more sense. Maybe I should update the code to make it test whether significantly differ from 1.

**Comment by bpbond** [comment]
Yes, that's definitely the test we want.
